src/models/Order.js

```diff
@@ -1,7 +1,10 @@
 const mongoose = require('mongoose');
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').config();
+}
```

Comment:
Why do we only need dotenv in non-production? aren't we fetching values from `process.env` in both environments? And also doesn't the dotenv get already loaded in `expressServer.js` when the app starts anyway?

Reply:
@akosasante sure! will commit changes now, just to make sure-  when we create express server in ### expressServer.js with `const app = express();`  and call it in **app.js** which gets executed by default when we run the server with _npm run dev_ -this action triggers everything related with **app**  (libraries, middlewares) so we don't need to require libraries and packages that we've already put in expressServer.js ? for examle everytime I was using environment variables in different modules- i was requring   `require('dotenv').config();`

Reply:
Yes that's correct. The only time you may need to add it is for scripts that you're running without starting up the express app. For example, the `fetchAlbumSpotify.js` script. Though even in that case you can require it via command-line flags instead if you prefer: `node -r dotenv/config fetchAlbumSpotify.js`

Reply:
@akosasante thank you!!!